src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -26756,5 +26756,79 @@ private void Method(IInterface inter, Func<(Type, Type), string> keySelector)
             var comp2 = CreateCompilation(source1, references: new[] { comp1.EmitToImageReference() }, options: TestOptions.DebugDll);
             CompileAndVerify(comp2);
         }
+
+        [Fact]
+        [WorkItem(1090920, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1090920")]
+        public void GetSymbolInfoOnInitializerOfValueTupleField()
+        {
+
+            var source = @"
+namespace System
+{
+    public struct ValueTuple<T1>
+    {
+        public T1 Item1 = default;
+        public ValueTuple(T1 item1) { }
+    }
+}
+";
+
+            var comp = CreateCompilation(new[] { source }, options: TestOptions.DebugDll);
+
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
+            var literal = tree.GetRoot().DescendantNodes().OfType<LiteralExpressionSyntax>().Single();
+            Assert.True(model.GetSymbolInfo(literal).IsEmpty);
+        }
+
+        [Fact, WorkItem(1090920, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1090920")]
+        public void NameofFixedBuffer()
```

Comment:
>NameofFixedBuffer [](start = 20, length = 17)

Is this test added to cover changes in MethodCompiler?

Reply:
Yes, it covers the handling in `if (fieldSymbol.IsFixedSizeBuffer && compilationState.Emitting)` that is now reachable.